Assets/Editor/EventEditors/CardsDealtGameEventEditor.cs

```diff
@@ -24,19 +24,19 @@ public class CardsDealtGameEventEditor : Editor
         companionId = EditorGUILayout.TextField(companionId);
         
         if (GUILayout.Button("Test Raise Event")) {
-            CompanionInstance companionInstance = CombatEntityManager.Instance
-                .getCompanionInstanceById(companionId);
-            // If you're getting a null pointer error here, it's probably because the id pasted
-            // into the field above isn't corresponding to an id the CombatEntityManager was made 
-            // aware of
-            CombatEntityInEncounterStats stats = companionInstance.stats;
-            InCombatDeck deckFrom = companionInstance.inCombatDeck;
-            if(cardInfo != null) {
-                cardsDealtEvent.Raise(new CardsDealtEventInfo(
-                    new List<Card>() { new Card(cardInfo) },
-                    companionInstance
-                ));
-            }
+            // CompanionInstance companionInstance = CombatEntityManager.Instance
```

Comment:
Delete, beauty of git is we can always revert :)

Reply:
Yeah this is the problem with the size of this commit, I couldn't be bothered to do a final git diff before pushing so I missed everything I commented out and forgot to come back and remove